lsif-java/src/main/scala/com/sourcegraph/lsif_java/commands/SnapshotLsifCommand.scala

```diff
@@ -59,7 +59,10 @@ case class SnapshotLsifCommand(
       in = AbsolutePath.of(inputPath, sourceroot)
       doc <- SnapshotLsifCommand.parseTextDocument(in, sourceroot)
     } {
-      SnapshotCommand.writeSnapshot(doc, finalOutput)
+      val docPath = AbsolutePath.of(Path.of(doc.getUri), sourceroot).toRealPath()
```

Comment:
To fix the Java 9 error
```suggestion
      val docPath = AbsolutePath.of(Paths.getdoc.getUri), sourceroot).toRealPath()
```
From CI
```
[error] /home/runner/work/lsif-java/lsif-java/lsif-java/src/main/scala/com/sourcegraph/lsif_java/commands/SnapshotLsifCommand.scala:62:42: value of is not a member of object java.nio.file.Path
[error]       val docPath = AbsolutePath.of(Path.of(doc.getUri), sourceroot).toRealPath()
[error]                                          ^
```